pkg/cmd/server/kubernetes/master_config.go

```diff
@@ -454,9 +454,14 @@ func DefaultOpenAPIConfig() *openapicommon.Config {
 			op := r.Operation
 			path := r.Path
 			//TODO/REBASE this is gross
+			// +1
 			if strings.HasPrefix(path, "/oapi/v1/namespaces/{namespace}/processedtemplates") {
 				op = "createNamespacedProcessedTemplate"
+			} else if strings.HasPrefix(path, "/apis/template.openshift.io/v1/namespaces/{namespace}/processedtemplates") {
+				op = "createNamespacedProcessedTemplateV1"
 			} else if strings.HasPrefix(path, "/oapi/v1/processedtemplates") {
+				op = "createProcessedTemplateForAllNamespacesV1"
+			} else if strings.HasPrefix(path, "/apis/template.openshift.io/v1/processedtemplates") {
 				op = "createProcessedTemplateForAllNamespaces"
 			} else if strings.HasPrefix(path, "/oapi/v1/namespaces/{namespace}/generatedeploymentconfigs") {
 				op = "generateNamespacedDeploymentConfig"
```

Comment:
the openapi generator will fail terribly without this (and I think master won't start as well..) i would love to get rid of this hack, but I don't have enough background knowledge.